apcd-cms/src/apps/view_users/views.py

```diff
@@ -25,61 +25,6 @@ def get_context_data(self, *args, **kwargs):
 
```

Comment:
Pass user_content in from main view by passing it in as an argument to the get_context_data function like this:

def get_context_data(self, user_content = user_content, *args, **kwargs):

Then remove line 23